src/components/btn.jsx

```diff
@@ -0,0 +1,9 @@
+export default function Button(props) {
```

Comment:
Als je export default gebruikt, zet je dit onderaan je bestand. Je kunt de normale export wel op deze plek gebruiken. 

```suggestion
function Button(props) {
         return(
        <>
            <button disabled={props.disabled}>
                {props.name}
            </button>
        </>
    )
}

export default Button;
```